eng/pipelines/templates/steps/common.yml

```diff
@@ -0,0 +1,9 @@
+steps:
```

Comment:
Should we call this step "npm setup" or something more descriptive then common?

Reply:
It now does a number of common things that every job should start with (it no longer just sets up NPM). I’m okay with leaving it as-is unless you feel strongly.

Reply:
sounds good I didn't know if the goal was everything common or just the npm setup. I'm fine keeping it named common.